cmd/internal/abidbbuilder/abidbbuilder.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 The go-ethereum Authors
```

Comment:
There's no need to make this an internal command and stash it deep away. You should put in into the `clef` folder, and add a build flag so it's not built. Wrt how to run it correctly, instead of manually screwing around, you can add a `go generate` statement to the clef.go file that will call this with the correct params. Then to pull a new fresh json, the user would only need to do `go generate ./cmd/clef`, and magic.

Reply:
sounds like magic, cool!

Reply:
> Then to pull a new fresh json, the user would only need to do go generate ./cmd/clef, and magic.

That's not the scenario, however... The 4byte lists are structured in a directory, where each file is a 4byte identifier containing one or multiple method signatures. So when you the thing to produce a json-file, you need to provide it with where to look. 

Another alternative is that I could remove it from go-ethereum and put it somewhere else. I initially thought it would reuse more parts of the abi and signer packages, but in the end only crypto was used.